### PR TITLE
Defining what we mean by meeting

### PR DESCRIPTION
**Pull request by ilyas1974**
Per our meeting this past week, I have made changes to our meeting guidance document.

### CODE REVIEW

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 ## Introduction
 In this new hybrid world there are many challenges ahead.  This is our current attempt as a team to navigate.  We know we're wrong at least in some ways, but will bravely move forward, learn, and then adjust as appropriate.
 
+## Definition
+To distinguish team gatherings that falls into the guidelines below from others, we should define what we mean by a meeting.
```
Comment (MattGal) [inline Documentation/TeamProcess/DevGuide/meetings.md:7]: nit: plurality agreement: 'that fall into'

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 2]
```diff
@@ -3,6 +3,11 @@
 ## Introduction
 In this new hybrid world there are many challenges ahead.  This is our current attempt as a team to navigate.  We know we're wrong at least in some ways, but will bravely move forward, learn, and then adjust as appropriate.
 
+## Definition
+To distinguish team gatherings that falls into the guidelines below from others, we should define what we mean by a meeting. 
+- A **Meeting** - for the context of our principles and guidelines below - is any gathering of the team where discussions or decisions are made that will long lived (more than a week).
```
Comment (MattGal) [inline Documentation/TeamProcess/DevGuide/meetings.md:8]: How do people who are having a conversation know this in advance?  There may be no good answer here, but maybe word it like 'where we plan to make decisions that will ...'

**Review by ilyas1974 — COMMENTED** [review]
Incorporating feedback

File: Documentation/TeamProcess/DevGuide/meetings.md (hunk 1 above)
Comment (ilyas1974) [reply Documentation/TeamProcess/DevGuide/meetings.md]: thank you. Changes made.

File: Documentation/TeamProcess/DevGuide/meetings.md (hunk 2 above)
Comment (ilyas1974) [reply Documentation/TeamProcess/DevGuide/meetings.md]: Thank you for the suggestion. Changes made.

**Review by missymessa — DISMISSED** [review]
LGTM shipit

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 3]
```diff
@@ -3,6 +3,11 @@
 ## Introduction
 In this new hybrid world there are many challenges ahead.  This is our current attempt as a team to navigate.  We know we're wrong at least in some ways, but will bravely move forward, learn, and then adjust as appropriate.
 
+## Definition
+To distinguish team gatherings that fall into the guidelines below from others, we should define what we mean by a meeting. 
+- A **Meeting** - for the context of our principles and guidelines below - is any gathering of the team where a planned discussion or decision is made that will long lived (more than a week). 
+- Team social events (such as the weekly Prague Coffee Break or the Redmond Happy Hour), Stand Up (the daily one in Prague, the twice weekly one in Redmond, and the daily FR one), and 1:1 do not meet our definition of a meeting.
```
Comment (jonfortescue) [inline Documentation/TeamProcess/DevGuide/meetings.md:9]: Weekly team sync probably also does not count

Comment (adiaaida) [reply Documentation/TeamProcess/DevGuide/meetings.md:9]: I think I actually disagree. Sometimes they tell us important things in the weekly team sync

**Review by markwilkie — DISMISSED** [review]
I like it - thanks Ilya

**Review by Chrisboh — APPROVED** [review]

**Pull request merged**